Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -348,8 +348,12 @@ void RenderTreeUpdater::updateElementRenderer(Element& element, const Style::Ele
     else
         element.clearDisplayContentsStyle();
 
-    if (!hasDisplayContents && !elementUpdateStyle.hasAutoLengthContainIntrinsicSize())
-        element.clearLastRememberedSize();
+    if (!hasDisplayContents) {
+        if (elementUpdateStyle.containIntrinsicWidthType() != ContainIntrinsicSizeType::AutoAndLength)
+            elementUpdateStyle.isHorizontalWritingMode() ? element.setLastRememberedLogicalWidth(LayoutUnit(-1)) : element.setLastRememberedLogicalHeight(LayoutUnit(-1));
+        if (elementUpdateStyle.containIntrinsicHeightType() != ContainIntrinsicSizeType::AutoAndLength)
+            elementUpdateStyle.isHorizontalWritingMode() ? element.setLastRememberedLogicalHeight(LayoutUnit(-1)) : element.setLastRememberedLogicalWidth(LayoutUnit(-1));
```

Comment:
Setting `-1` will force the creation of the rare data.

Reply:
Done